sale_order_import_ubl/wizard/sale_order_import.py

```diff
@@ -111,10 +134,10 @@ def parse_ubl_sale_order(self, xml_root):
             company_dict = {"vat": company_dict_full["vat"]}
         delivery_xpath = xml_root.xpath("/%s/cac:Delivery" % root_name, namespaces=ns)
         shipping_dict = {}
-        delivery_dict = {}
+        commitment_date = False
         if delivery_xpath:
             shipping_dict = self.ubl_parse_delivery(delivery_xpath[0], ns)
-            delivery_dict = self.ubl_parse_delivery_details(delivery_xpath[0], ns)
+            commitment_date = self._get_order_commitment_date(delivery_xpath[0], ns)
```

Comment:
I don't understand this change, `ubl_parse_delivery_details` has been merged into 13.0 https://github.com/OCA/edi/pull/349/files
The commit is present in the 14 branch but it does not have all its content. Indeed the part that was meant for `base_ubl` has been left aside.
This is why we have this change now.

In my opinion this commit  should be properly recovered https://github.com/OCA/edi/pull/349/commits

Reply:
@phuctranerp Will you please check  above comments ?

Reply:
Good day @TDu, Thanks for your review. The 13.0 PR has been brought forward to 14.0 by https://github.com/OCA/edi/pull/406. So this PR is currently depending on it. Please help to review both PR. Thank you in advance.